nwb_conversion_tools/baserecordingextractorinterface.py

```diff
@@ -20,16 +20,17 @@ def __init__(self, **input_args):
         self.recording_extractor = self.RX(**input_args)
 
     def get_metadata_schema(self):
-        metadata_schema = get_base_schema()
+        # Initiate empty metadata schema
+        metadata_schema = get_metadata_schema()
 
-        # ideally most of this be automatically determined from pynwb docvals
-        metadata_schema['properties'].update(
+        # Initiate Ecephys metadata
+        metadata_schema['properties']['Ecephys'] = dict()
+        metadata_schema['properties']['Ecephys'].update(
             Device=get_schema_from_hdmf_class(Device),
             ElectrodeGroup=get_schema_from_hdmf_class(ElectrodeGroup),
             ElectricalSeries=get_schema_from_hdmf_class(ElectricalSeries)
         )
-        required_fields = ['Device', 'ElectrodeGroup', 'ElectricalSeries']
-        metadata_schema['required'] += required_fields
+        metadata_schema['properties']['Ecephys'] = ['Device', 'ElectrodeGroup', 'ElectricalSeries']
```

Comment:
@CodyCBakerPhD could you please check this? Its likely important for the Converters using BaseRecordingExtractorInterface